CHANGELOG.rst

```diff
@@ -15,13 +15,34 @@ Change Log
 Unreleased
 ~~~~~~~~~~
 
+[3.8.0] - 2020-08-31
+~~~~~~~~~~~~~~~~~~~~
+
+Updated
+_______
+
+* Renamed "custom metric" to "custom attribute" throughout the monitoring library. This decision can be read about in the ADR 0002-custom-monitoring-language.rst.  The following have been deprecated:
+
+  * set_custom_metric (use set_custom_attribute)
+  * set_custom_metrics_for_course_key (use set_custom_attributes_for_course_key)
+  * MonitoringCustomMetricsMiddleware (use CachedCustomMonitoringMiddleware)
+  * CachedCustomMonitoringMiddleware.accumulate_metric (use CachedCustomMonitoringMiddleware.accumulate_attribute)
+
+    * This wasn't meant to be used publicly, but was deprecated just in case.
+
```

Comment:
FYI: @timmc-edx: I added a note about this as a reminder for removing the code later.